src/mat_dense_complex.c

```diff
@@ -2,7 +2,7 @@
 
 #include <math.h>
 
-#include <openblas/lapacke.h>
+#include <lapacke.h>
```

Comment:
Try just removing this line? Looks like LAPACK should get pulled in when `bf/blas.h` in included.